Added rotation calculation

atmadsen1 opened the pull request:
added orientation of robot calculation (A side only) and fixed some spacing issues with code.

atmadsen1 committed:
Update navigation.py

JadeTheGecko commented:
Andy,

What is variable l? Please make that more descriptive.

atmadsen1 commented:
Variable L is the distance from the rightmost ball to the robot

> On Feb 3, 2019, at 3:28 AM, Kyle Melton <notifications@github.com> wrote:
> 
> Andy,
> 
> What is variable l? Please make that more descriptive.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

atomlinson94 commented:
Alright, then please add a comment to the code or make the variable name more descriptive.

atmadsen1 committed:
added things

myAlgorithm is a* on crack.
run simulation as main and it will allow you to place start/end points and obstacles. press start to run the algorithm

cnnyra committed:
Merge branch 'master' into master
